nestkernel/conn_builder.cpp

```diff
@@ -241,13 +250,22 @@ nest::ConnBuilder::ConnBuilder( NodeCollectionPTR sources,
 
 nest::ConnBuilder::~ConnBuilder()
 {
-  delete weight_;
-  delete delay_;
+  for ( auto weight : weight_ )
```

Comment:
Looking at this, maybe variable names should be plural `weights_` and `delays_`?